src/SFA.DAS.EmployerIncentives.Web/Authorisation/EmployerAccountAuthorizationHandler.cs

```diff
@@ -25,6 +34,11 @@ private Task<bool> IsEmployerAuthorised(AuthorizationHandlerContext context)
                 return Task.FromResult(false);
             }
 
+            if (_configuration.AchieveServiceBaseUrl.Contains(mvcContext.HttpContext.Request.Host.ToString()))
```

Comment:
What's this?

Reply:
The redirect to the Achieve Service URL occurs within the controller context of the HomeController. This prevents an exception being thrown from an acceptance test because the routing values are invalid.